managed/services/telemetry/datasources.go

```diff
@@ -30,6 +30,9 @@ import (
 // DataSourceName data source name.
 type DataSourceName string
 
+//go:generate ../../../bin/mockery -name=DataSourceLocator -case=snake -inpkg -testonly
+//go:generate ../../../bin/mockery -name=DataSource -case=snake -inpkg -testonly
+
```

Comment:
nitpick: we usually define a `deps.go` file to contain all the interfaces and mockery directives for a service, is there any reason we are not following this approach here? if no reason, then I think it would be better to move them to a `deps.go` file to maintain consistency across the codebase.

Reply:
Makes sense

Created deps.go file